pkg/trace/sampler/sampler.go

```diff
@@ -163,6 +169,14 @@ func getMetric(s *pb.Span, k string) (float64, bool) {
 	return val, ok
 }
 
+func getMeta(s *pb.Span, k string) (string, bool) {
```

Comment:
It might make sense to put this into `pkg/trace/traceutil/span.go` next to [`SetMeta`](https://github.com/DataDog/datadog-agent/blob/f1403b72dd7417c30a73bdbbf0edf83e6b0ffcf3/pkg/trace/traceutil/span.go#L50) but I don't feel strongly on it and I'd be fine if you decide to leave it here.